pkg/api/unversioned/group_version.go

```diff
@@ -46,6 +46,17 @@ func (gr *GroupResource) String() string {
 	return gr.Resource + "." + gr.Group
 }
 
+// ParseGroupResource turns "resource.group" string into a GroupResource struct. It reports error
+// if it cannot parse the string.
```

Comment:
> Missing the error case?

I wasn't convinced that I wanted to fail, but if we want to we can.  I could see value in having an unambiguously empty parsing ability.  It's something I'd like to have eventually for `GroupVersion` once we're past the legacy stage.